src/backend/launcher.ts

```diff
@@ -567,34 +568,37 @@ async function callRunner(
   return new Promise<ExecResult>((res, rej) => {
     const child = spawn(bin, commandParts, {
       cwd: runner.dir,
-      env: { ...process.env, ...options?.env }
+      env: { ...process.env, ...options?.env },
+      signal: abortController.signal
     })
 
     const stdout: string[] = []
     const stderr: string[] = []
 
-    child.stdout.on('data', (data: Buffer) => {
+    child.stdout.setEncoding('utf-8')
+    child.stdout.on('data', (data: string) => {
       if (options?.logFile) {
-        appendFileSync(options.logFile, data.toString())
+        appendFileSync(options.logFile, data)
       }
 
       if (options?.onOutput) {
-        options.onOutput(data.toString(), child)
+        options.onOutput(data)
       }
 
-      stdout.push(data.toString().trim())
+      stdout.push(data.trim())
     })
 
-    child.stderr.on('data', (data: Buffer) => {
+    child.stderr.setEncoding('utf-8')
+    child.stderr.on('data', (data: string) => {
       if (options?.logFile) {
-        appendFileSync(options.logFile, data.toString())
+        appendFileSync(options.logFile, data)
```

Comment:
Being able to call `kill` on the `ChildProcess` isn't the only reason we're passing it here. It's also useful if you have to communicate with it, like I'm doing in #1887 ([exact lines if you're curious](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/cbee1e4c2e58f771315270c8a57e8426403d06db/src/backend/save_sync.ts#L57-L59)). Please don't remove it.

Reply:
I don't get it, what are we removing? The `.toString()`?